packages/jest-worker/src/workers/__tests__/processChild.test.ts

```diff
@@ -24,8 +15,29 @@ import {
   PARENT_MESSAGE_OK,
 } from '../../types';
 
-let ended;
-let mockCount;
+const spyProcessSend = jest.spyOn(process, 'send');
+
+declare global {
```

Comment:
Without this declaration git thinks that this file is completely new. See second commit.

Perhaps let’s keep it for now and I will remove it in #13416